scripts/generate

```diff
@@ -609,6 +609,27 @@ ${content}
 </html>
 `));
 
+//    program :: Array String -> Future String String
+const program = pipe ([
+  array (reject ('Missing command-line argument'))
+        (compose (K) (resolve)),
+  map (curry2 (path.join)),
+  chain (path =>
+           pair (lift3 (toDocument))
+                (bimap (chain (compose (eitherToFuture)
+                                       (field ($.String) ('version'))))
+                       (chain (compose (eitherToFuture)
+                                       (field ($.String) ('description'))))
+                       (join (Pair)
```

Comment:
It took me a minute to wrap my head around what's happening here, but it's actually just a point free version of `f (g x) (h x)`. You duplicate (`join`) `x` into a `Pair`, then then `bimap` `g` and `h` over the values in the Pair, and finally combine into `f` using `pair`.

I think it's a nice approach to the problem. I also find myself in need of a solution to this quite a lot. Ramda has [`converge`](https://ramdajs.com/docs/#converge), and I was never able to find a straight-forward solution with Sanctuary. This comes the closest I've seen. :)

Reply:
Actually, `converge (f, [g, h])` is just `lift2 (f) (g) (g)`! So this code could be rewritten like:

```js
lift2 (lift3 (toDocument))
      (chain (compose (eitherToFuture) (field ($.String) ('version'))))
      (chain (compose (eitherToFuture) (field ($.String) ('description'))))
```

Reply:
I updated the [common combinators table](https://gist.github.com/Avaq/1f0636ec5c8d6aed2e45) to include the `converge` combinator, which is `lift2` in Sanctuary. I also updated the Ramda to Sanctuary Wiki to show that `lift<n>` can be used to replace `R.converge`.

Reply:
`lift2` is `apply2way`?! :exploding_head:

Reply:
I know, right! :D

Reply:
#92